bad_exclusions_list.py

```diff
@@ -0,0 +1,112 @@
+bad_exclusions_list = [
+    "THIS IS NOT A LIST OF EXCLUSIONS IN YOUR ENVIRONMENT, BUT A LIST OF EXCLUSIONS WE RECOMMEND AGAINST!",
+    "For the most up to date list and additional information refer to",
+    "https://www.cisco.com/c/en/us/support/docs/security/amp-endpoints/213681-best-practices-for-amp-for-endpoint-excl.html",
+    "",
+    "AcroRd32.exe",
+    "addinprocess.exe",
+    "addinprocess32.exe",
+    "addinutil.exe",
+    "bash.exe",
+    "bginfo.exe",
+    "bitsadmin.exe",
+    "cdb.exe",
+    "csi.exe",
+    "dbghost.exe",
+    "dbgsvc.exe",
+    "dnx.exe",
+    "dotnet.exe",
+    "excel.exe",
+    "fsi.exe",
+    "fsiAnyCpu.exe",
+    "iexplore.exe",
+    "java.exe",
+    "kd.exe",
+    "lxssmanager.dll",
+    "msbuild.exe",
+    "mshta.exe",
+    "ntkd.exe",
+    "ntsd.exe",
+    "outlook.exe",
+    "psexec.exe",
+    "powerpnt.exe",
+    "powershell.exe",
+    "rcsi.exe",
+    "svchost.exe",
+    "schtasks.exe",
+    "system.management.automation.dll",
+    "windbg.exe",
+    "winword.exe",
+    "wmic.exe",
+    "wuauclt.exe",
+    ".7z",
+    ".bat",
+    ".bin",
+    ".cab",
+    ".cmd",
+    ".com",
+    ".cpl",
+    ".dll",
+    ".exe",
+    ".fla",
+    ".gif",
+    ".gz",
+    ".hta",
+    ".inf",
+    ".java",
+    ".jar",
+    ".job",
+    ".jpeg",
+    ".jpg",
+    ".js",
+    ".ko",
+    ".ko.gz",
+    ".msi",
+    ".ocx",
+    ".png",
+    ".ps1",
+    ".py",
+    ".rar",
+    ".reg",
+    ".scr",
+    ".sys",
+    ".tar",
+    ".tmp",
+    ".url",
+    ".vbe",
+    ".vbs",
+    ".wsf",
+    ".zip",
+    "bash",
+    "java",
+    "python",
+    "python3",
+    "sh",
+    "zsh",
+    "/",
+    "/bin",
+    "/sbin",
+    "/usr/lib",
+    "C:",
+    "C:/",
+    "C:/*",
+    "D:/",
+    "D:/*",
+    "C:/Program Files/Java",
+    "C:/Temp/",
+    "C:/Temp/*",
+    "C:/Users/",
+    "C:/Users/*",
+    "C:/Windows/Prefetch",
+    "C:/Windows/Prefetch/",
+    "C:/Windows/Prefetch/*",
+    "C:/Windows/System32/Spool",
+    "C:/Windows/System32/CatRoot2",
+    "C:/Windows/Temp",
+    "C:/Windows/Temp/",
+    "C:/Windows/Temp/*",
+    "C:/Program Files/<company name>/",
+    "C:/Program Files (x86)/<company name>/",
+    "C:/Users/<UserProfileName>/AppData/Local/Temp/",
+    "C:/Users/<UserProfileName>/AppData/LocalLow/Temp/"
```

Comment:
Wouldn't these exclusions show as C:\\Users (double-backslashes, not forward slashes?)

Reply:
Changed these to \\